### DISCUSSION TITLE
Nextcloud/webdav storage adapter

### DISCUSSION THREAD

**Discussion by cbertelli** [directus/directus#14361] [Feature Requests]
It could be very useful for a project I'm working on, but there could be a more general interest in it.

**Comment by Silbad** [comment]
Hello, if I develop the webdav adapter myself, is it feasible?

**Reply by rijkvanzanten** [reply]
Sure! You could take a look at one of the core `@directus/storage-driver-X` packages in this repo as a starting point. As long as you implement every method as per the type definitions it should work 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
